### ISSUE TITLE
Bug when Lycan mob strikes player, crashes client.

### ISSUE THREAD

**Issue by sharkck** [Lycanite/LycanitesMobs#188]
[crash-2016-11-15_06.39.27-client.txt](https://github.com/Lycanite/LycanitesMobs/files/591404/crash-2016-11-15_06.39.27-client.txt)
See attached.

I've experienced this several times now and so has a few other players.

**Comment by Lycanite** [comment]
Thanks for the bug report, there are a few crashes like this happening and I hope to have them fixed soon.

**Comment by P3rf3ctXZer0** [comment]
Here is a full log! 
https://gist.github.com/P3rf3ctXZer0/38f6931b09b8081af51cb0c42eb0df8e

and

Crash Log
https://gist.github.com/P3rf3ctXZer0/fdd4611ed89e9471ff78061c9033a615

**Comment by sharkck** [comment]
As an update/FYI, I updated everything to Forge 2147 and the problem still exists.  Thank you for your attention on this matter!  Hope to see an update soon!

**Comment by xlxAciDxlx** [comment]
The casting bugs are a known issue.

**Comment by Lycanite** [comment]
I'll try and get these bugs sorted soon, unfortunately I've been busy with things lately but hope to have this sorted soon!

**Comment by sharkck** [comment]
Not meaning to be a bother or to rush you.  Is there an ETA on when to expect the next update?  Just really excited and anticipating the update :)

**Comment by Lycanite** [comment]
No probs! I hope to have this sorted soon, I really do need to get a new update out XD Then there's Minecraft 1.11 to update to lol! I'm also working on a mod for ARK: Survival Evolved so I'll need to make sure that I pry myself away from that and give Lycanites Mobs some love!

**Comment by Lycanite** [comment]
I'm also having a PC crisis as my main PC's Motherboard is faulty so I'm waiting on that to be repaired or replaced, in the meantime I'm using older, slower, hardware. :/

**Comment by xlxAciDxlx** [comment]
Is there any chance there's a way for someone else to do a PR to fix the casting bugs? I wouldn't mind trying to help, but as far as I can tell there's no way to properly get the source (plus no build.gradle for any dependencies)

**Comment by Lycanite** [comment]
I'm going try and get an update out later today, to get the source you can use git to pull from here.

**Comment by xlxAciDxlx** [comment]
Sorry, not trying to rush you. I just figured I'd try to help you out if you were busy/having problems. :)

**Comment by Lycanite** [comment]
Lol no probs, I've got some spare time later today now so all is good, I'd like to make more progress on some new mobs too.

**Comment by Lycanite** [comment]
I'm closing these duplicate casting issues, I may have a solution for this coming in the next update 1.15.1.5! :D

**Comment by sharkck** [comment]
Thank you very much for the update!!!  It's so nice to be able to play crash-free!  You're awesome!

**Comment by Lycanite** [comment]
No problem! If the issue occurs again just post a new issue. I fear that projectiles may still cause crashes, but we shall see! I think it's an underlying issue with either Minecraft itself or Forge, but I'm able to suppress the crashes, it just means that sometimes the client ignores an update from the server.
